test/typings.ts

```diff
@@ -22,23 +22,23 @@ interface Dependencies {
   func(value: string): string;
 }
 
-const epic1: Epic<FluxStandardAction, State> = (action$, store) =>
+const epic1: Epic<FluxStandardAction, FluxStandardAction, State> = (action$, store$) =>
   action$.pipe(
     ofType('FIRST'),
     mapTo({
       type: 'first',
-      payload: store.getState()
+      payload: store$.getState()
```

Comment:
We probably shouldn't test for getState anymore. Can we change this to something like

```ts
const epic1: Epic<FluxStandardAction, FluxStandardAction, State> = (action$, state$) =>
  action$.pipe(
    ofType("FIRST"),
    withLatestFrom(state$),
    map(([, state]) => ({
      type: "first",
      payload: state
    }))
  );
```

Reply:
@evertbouw since `getState` is not removed should we keep it? I wouldn't be opposed to adding a test for the `state$` stream too, though. Would you want that as part of this PR? It's not really related to the changes.

Reply:
Yes fair enough. It should change soon but it doesn't have to be part of this PR